lib/reverse_sentence.rb

```diff
@@ -1,6 +1,25 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2) n = my_sentence.length
+# Space complexity: O(1) only one array gets made of that sentence will ever be made
 def reverse_sentence(my_sentence)
```

Comment:
This time complexity is O(n) since you only loop through the list of words once.  It also has space complexity of O(n) since you make an array `my_sentence.split`.